src/redux/reducers/experiments/experimentsLoaded.js

```diff
@@ -0,0 +1,27 @@
+const experimentsLoaded = (state, action) => {
+  const { experiments } = action.payload;
+
+  const newExperiments = experiments.reduce((acc, curr) => ({
+    ...acc,
+    ids: [...acc.ids, curr.experimentId],
+    [curr.experimentId]: {
+      projectUuid: curr.projectId,
+      name: curr.experimentName,
+      description: curr.description,
+      id: curr.experimentId,
+      createdAt: curr.createdAt,
+      lastViewed: curr.lastViewed,
+    },
+  }), { ids: [...state.ids] });
+
+  return {
+    ...state,
+    ...newExperiments,
+    meta: {
+      ...state.meta,
+      loading: false,
```

Comment:
should we set `error: false` too? it is what we usually do when we set `load: false`, right?

Reply:
Error should be reset if the data is reloaded, which involves setting up the loading state again. `error: false` is set in `experimentsLoading`.